python_modules/dagster/dagster/_core/definitions/automation_tick_evaluation_context.py

```diff
@@ -74,6 +75,7 @@ def __init__(
         self._materialize_run_tags = materialize_run_tags
         self._observe_run_tags = observe_run_tags
         self._auto_observe_asset_keys = auto_observe_asset_keys or set()
+        self._partition_loading_context = self._evaluator.asset_graph_view.partition_loading_context
```

Comment:
i think adding the property on asset graph view is probably fine, but would be easy to avoid too if we wanted to call into the instance queryer or something

Reply:
yeah I think my overall goal is to delete the instance queryer in the long run, so I want to avoid referencing it in new code paths.

also considering making the AssetGraphView a context manager and having the idiomatic thing to be:

```python
with AssetGraphView(...):
    ...
```

and have that set up the partition loading context, but want to make that change separately from this